iroh-net/src/dns/node_info.rs

```diff
@@ -16,32 +44,34 @@ use url::Url;
 
 use crate::{key::SecretKey, AddrInfo, NodeAddr, NodeId};
 
-/// The DNS name for the iroh TXT record
+/// The DNS name for the iroh TXT record.
 pub const IROH_TXT_NAME: &str = "_iroh";
 
-/// The attributes supported by iroh for `_iroh` DNS records
+/// The attributes supported by iroh for `_iroh` DNS resource records.
+///
+/// The resource record uses the lower-case names.
 #[derive(
     Debug, strum::Display, strum::AsRefStr, strum::EnumString, Hash, Eq, PartialEq, Ord, PartialOrd,
 )]
 #[strum(serialize_all = "kebab-case")]
 pub enum IrohAttr {
-    /// `relay`: URL of home relay
+    /// URL of home relay.
     Relay,
-    /// `addr`: Direct address
+    /// Direct address.
     Addr,
 }
 
-/// Lookup node info by domain name
+/// Looks up node info by DNS name.
 ///
-/// The domain name must either contain an _iroh TXT record or be a CNAME record that leads to
-/// an _iroh TXT record.
-pub async fn lookup_by_domain(resolver: &TokioAsyncResolver, domain: &str) -> Result<NodeAddr> {
-    let attrs = TxtAttrs::<IrohAttr>::lookup_by_domain(resolver, domain).await?;
+/// The resource records returned for `name` must either contain an `_iroh` TXT record or be
+/// a CNAME record that leads to an `_iroh` TXT record.
```

Comment:
Even if unlikely to change here I think it's clearer to reference `IROH_TXT_NAME` here

Reply:
good call, i've replaced these with the constant.